pkg/ebpf/tracee.go

```diff
@@ -815,7 +832,7 @@ func (t *Tracee) getOptionsConfig() uint32 {
 	return cOptVal
 }
 
-func (t *Tracee) computeConfigValues() []byte {
+func (t *Tracee) computeConfigValues(newPolicies *policy.Policies) []byte {
```

Comment:
I think this function could (and should) be moved to the policy snapshots.

Reply:
Well, the problem is this would reinject the bpf logic into the Policies/Snapshots types, which is what we're trying to avoid: coupling things together.

Reply:
I think that the existence of `UpdateBPF` in snapshots show that the snapshot object is the coupling point. That's not a bad thing, there must be some point of coupling if we want to make policies meaningful in bpf logic.
Maybe, the `UpdateBPF` function should be taken out of the snapshots object so that the coupling will be out of the object and into some "effect function".

Reply:
@NDStrahilevitz after analyse this better I agree with you. We really need to split logic and diminish the tracee file. However, the complexity involved is also huge. For example, the `computeConfigValues` takes care of "config_map" which has a lot from policies but it's a tracee generic config place. And splitting or moving its logic can compromise the configVal `config_entry` encode understanding. What I propose now is to update this [EPIC](https://github.com/aquasecurity/tracee/issues/3239) with next steps, so we can tackle then meanwhile.